setup.py

```diff
@@ -12,7 +13,7 @@
     download_url='https://github.com/D-E-F-E-A-T/Selenium-Cookie-Injector/archive/v_01.tar.gz',
     keywords=['SELENIUM', 'COOKIE'],
     install_requires=[
-        'pyaes', 'pbkdf2', 'keyring', 'lz4', 'configparser'
+        'setuptools', 'wheel', 'pyaes', 'pbkdf2', 'keyring', 'lz4', 'configparser'
```

Comment:
You can remove setuptools and wheel from here because you are already installing in actions.